http/src/main/resources/init-resources/init-actions.sh

```diff
@@ -210,6 +212,18 @@ END
       echo "" > /etc/google_application_credentials.env
     fi
 
+    # Install RStudio license file, if specified
+    if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
```

Comment:
if rstudio is not gonna use spark, we probably don't need this?

Reply:
We currently support RStudio on Dataproc so I think it makes sense to install the license too.. we could decide in the future to remove RStudio support from Dataproc. Although maybe there are some R tool which use Spark 🤷‍♂️

Reply:
I see...UI still needs to make decision which runtime to create, are we creating GCE VMs for RStudio by default?

Reply:
Yeah, IIRC it defaults to GCE VM but the user can still choose to use Spark